Headers/x11/XGServer.h

```diff
@@ -60,6 +60,7 @@ typedef enum {
   Window	    grabWindow;
   struct XGGeneric  generic;
   id                inputServer;
+  int               randrEventBase, randrErrorBase;
```

Comment:
I personally prefer to have (instance) variables defined on separate lines.

Reply:
No problem. Done.